### DISCUSSION TITLE
Cannot download older versions of Security Onion

### DISCUSSION THREAD

**Discussion by pacosilva93** [Security-Onion-Solutions/securityonion#6981] [Unsupported Versions] [answered]
Hello, I am attempting to install Security Onion 16.4.6.6. Every time I click the link it takes me to the most updated version and requires 99 GBS to operate. This would take up a lot of space on my PC. Based on my instructor, this is the wrong method. 

Where can I download the older Onion version? Please help.

**Comment by phil1090** [comment, marked as answer]
Security Onion 16.04 has reached end of life: https://blog.securityonion.net/2021/04/security-onion-1604-has-reached-end-of.html
We no longer distribute it. If your class specifically requires 16.04, you may want to ask your instructor if they have a copy of the ISO available for you. Note that it is possible to install Security Onion 2 in Import Mode with minimal requirements. For more information, please see: https://docs.securityonion.net/en/2.3/first-time-users.html
Best of luck to you.
Phil

**Reply by pacosilva93** [reply]
Thank you for the response! Yeah that's odd since the tutorial was created in Aug '21 and the version in that tutorial looks far different than the most recent version. I will be letting my Prof know that this needs to be updated. 

I figured that they would provide the .ISO but its not in the materials for class.

**Reply by tomjones1977** [reply]
Sounds like lazy writing by the instructor.  We used to teach on 16.04, and went through the work to update all our teaching packs and our virtual environment when SO2 came out.
